cvss/tests/base.rs

```diff
@@ -5,6 +5,7 @@
 use core::str::FromStr;
 
 /// CVE-2013-1937
+#[cfg(feature = "default")]
```

Comment:
These should all be...

```suggestion
#[cfg(feature = "v3")]
```

...but other than that this looks good, thanks

Reply:
Note my comment only fixed one of them: there are several more that should all be `v3`

Reply:
This will case an error of:
```
error[E0599]: no method named `score` found for struct `Base` in the current scope
   --> cvss/tests/base.rs:274:21
    |
274 |     assert_eq!(base.score().value(), 1.8);
    |                     ^^^^^ method not found in `Base`
```

But I'll set them to `v3` and `std`